modules/data-streams/src/javaRestTest/java/org/elasticsearch/datastreams/logsdb/qa/StandardVersusLogsIndexModeChallengeRestIT.java

```diff
@@ -53,7 +53,7 @@
  * This test uses simple mapping and document structure in order to allow easier debugging of the test itself.
  */
 public class StandardVersusLogsIndexModeChallengeRestIT extends AbstractChallengeRestTest {
-    private final int numShards = randomBoolean() ? randomIntBetween(2, 5) : 0;
+    private final int numShards = randomBoolean() ? randomIntBetween(2, 4) : 0;
```

Comment:
Without this tweak test cluster sometimes goes out of memory due to the size of translog. I assume this is just due to large documents.